imgui-SFML.cpp

```diff
@@ -549,6 +549,15 @@ void SetLStickYAxis(sf::Joystick::Axis lStickYAxis, bool inverted) {
 }  // end of namespace SFML
 
 /////////////// Image Overloads
+void Image(const sf::Sprite& sprite) {
+    sf::FloatRect bounds = sprite.getLocalBounds();
+    sf::Vector2f scale = sprite.getScale();
+    sf::Color tintColor = sprite.getColor();
+    sf::Color borderColor = sf::Color::Transparent;
+    ImGui::SetCursorPos(ImVec2(sprite.getPosition().x - sprite.getOrigin().x * sprite.getScale().x, sprite.getPosition().y - sprite.getOrigin().y * sprite.getScale().y));
```

Comment:
I don't like how this gets repeated for all overloads - please make a helper function
Also, you can just do something like this:
```cpp
sf::Vector2f cursorPos = sprite.getPosition() - sprite.getOrigin() * sprite.getScale();
ImGui::SetCursorPos(ImVec2(cursorPos.x, cursorPos.y));
```